### PR TITLE
Refresh nuget topic

### PR DESCRIPTION
**Pull request by NextTurn**
### Thank you for contributing! Please confirm this pull request meets the following requirements:

- [x] I followed the contributing guidelines: https://github.com/github/explore/blob/master/CONTRIBUTING.md
- [x] I have no affiliation with the project I am suggesting (as a maintainer, creator, contractor, or employee).

### Which change are you proposing?

  - [x] Suggesting edits to an existing topic or collection
  - [ ] Curating a new topic or collection

---------------------------------------------------------------------

### Editing an existing topic or collection

I'm suggesting these edits to an existing topic or collection:
- [ ] Image (and my file is `*.png`, square, dimensions 288x288)
- [x] Content (and my changes are in `index.md`)

### CODE REVIEW

File: topics/nuget/index.md [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
+related: dotnet
+released: October 5, 2010
+short_description: NuGet is the package manager for .NET.
 topic: nuget
-url: https://www.nuget.org/
+url: https://www.nuget.org
 wikipedia_url: https://en.wikipedia.org/wiki/NuGet
 ---
-NuGet is a free and open source package manager designed for the Microsoft development platform. It is formerly known as NuPack.
+NuGet is the package manager for .NET. The NuGet client tools provide the ability to produce and consume packages. The NuGet Gallery is the central package repository used by all package authors and consumers.
```
Comment (leecow) [inline topics/nuget/index.md]: NuGet is used for more than just .NET. Parts. Including "designed for the Microsoft development platform" would be more accurate.

Comment (NextTurn) [reply topics/nuget/index.md]: I have no idea. The Nuget [website](https://www.nuget.org/policies/About) and [docs](https://docs.microsoft.com/nuget/what-is-nuget) say Nuget is the Microsoft-supported mechanism for sharing code for .NET.

Comment (NextTurn) [reply topics/nuget/index.md]: @leecow I've made the changes. But I still want to know which other platforms it supports.

**Comment by kolappannathan** [comment]
Hi,

I was the person who added the NuGet topic. Your changes completely remove the info that NuGet is open source.

**Comment by NextTurn** [comment]
Actually everything on GitHub is open source.

**Comment by kolappannathan** [comment]
@NextTurn Mostly yes but not always. I have seen organisation having GitHub repositories just for issue managements, docs, websites, etc...

It is a small detail but I feel it is an important one.

**Pull request merged**